patches/api/0006-Adventure.patch

```diff
@@ -3255,11 +3255,13 @@ index 9866c07c999f46cb585709804aaad710c3031d5a..c7c45e2de8cca1bf8b8e12752e08db62
          return message;
      }
  
-@@ -95,16 +96,66 @@ public class AsyncPlayerPreLoginEvent extends Event {
-      *
-      * @param message New kick message
-      */
--    public void setKickMessage(@NotNull final String message) {
++    /**
++     * Sets the kick message to display if getResult() != Result.ALLOWED
++     *
++     * @param message New kick message. At the point when this event is fired,
++     *     the server does not have the player's locale, so the translate component
++     *     will render with default locale({@link java.util.Locale#US}).
```

Comment:
The longer description should be in the main body of the javadoc.

Also it should be, 
```
When this event is fired, the player's locale is not
available. Therefore, any translatable component will be
rendered with the default locale, {@link java.util.Locale#US}.
<p>
Consider rendering any translatable yourself with {@link GlobalTranslator#render}
if the client's language is known.
```